tardis/model/base.py

```diff
@@ -376,16 +393,43 @@ def density(self):
         return self.model_state.density
 
     @property
-    def abundance(self):
-        if not self.raw_isotope_abundance.empty:
-            self._abundance = self.raw_isotope_abundance.decay(
-                self.time_explosion
-            ).merge(self.raw_abundance)
-        abundance = self._abundance.loc[
-            :, self.v_boundary_inner_index : self.v_boundary_outer_index - 1
+    def isotope_abundance(self):
+        raw_abundance = self.raw_abundance.copy()
+        raw_abundance["mass_number"] = -1
+        raw_abundance = raw_abundance.set_index(
+            [raw_abundance.index, "mass_number"]
+        )
+
+        raw_abundance.columns = pd.RangeIndex(start=0, stop=len(raw_abundance.columns), step=1)
+
+        if not self._raw_isotope_abundance.empty:
+            self._abundance = raw_abundance.append(
+                self._raw_isotope_abundance.decay(self.time_explosion)
+            )
+            norm_factor = self._abundance.sum(axis=0)
+            self._abundance /= norm_factor
+        else:
+            self._abundance = raw_abundance
```

Comment:
This is a small replacement for the merge method [here](https://github.com/tardis-sn/tardis/blob/c70594674b4e0998f3ea8a349911c1096e877331/tardis/io/decay.py#L112).
I could clean it up if this method is alright.